src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/jobsboard/api/controller/jobs/ExpressionsOfInterestPutShould.kt

```diff
@@ -0,0 +1,87 @@
+package uk.gov.justice.digital.hmpps.jobsboard.api.controller.jobs
+
+import org.junit.jupiter.api.Test
+import org.springframework.http.HttpStatus.CREATED
+import org.springframework.http.HttpStatus.NOT_FOUND
+import org.springframework.http.HttpStatus.OK
+
+class ExpressionsOfInterestPutShould : ExpressionsOfInterestTestCase() {
+  @Test
+  fun `create expression-of-interest with valid job-ID and prisoner's prison-number, when it does NOT exist`() {
```

Comment:
We can use the natural English language here. For example:
`create an expression of interest with a valid Job ID and prisoner's prisonNumber`
I tend to start with capital when I'm referring to an entity of my domain (Job, Employer) and camel case when I'm referring to a specific attribute (createdAt, prisonNumber). I make an exception with entity IDs, I try to always leave the entity name clean, followed by ID.

The result is less use of the dash '-', which introduces a new class of elements. I make the language as simple as possible with a few elements: Entities, attributes and plain English language.

Reply:
updated, removing all those hyphen/dash